OrPinhas_and_AvivShabtay_Project/OrPinhas_and_AvivShabtay/OrPinhas_and_AvivShabtay/Pool.h

```diff
@@ -10,21 +10,21 @@ class Pool
 	char* openingHours;
 
 public:
-	Pool(int isHavingEntertainment, char* strHours) 
+	Pool(int isHavingEntertainment, char* strHours)
 	{
-	openingHours = new char[strlen(strHours) + 1];
-	strcpy(openingHours, strHours);
-	this->isHavingEntertainment = isHavingEntertainment;
+		openingHours = new char[strlen(strHours) + 1];
+		strcpy(openingHours, strHours);
+		this->isHavingEntertainment = isHavingEntertainment;
 	};
```

Comment:
Add `this` before class member in order to distinguish between parameter and the class member himself.